en/payloads/README.md

```diff
@@ -4,24 +4,37 @@ PX4 supports a wide range of payloads and cameras.
 
 ## Mapping Drones
 
-* [Camera triggering](../peripherals/camera.md) via GPIO out
-* [Camera triggering](../peripherals/camera.md) via PWM out
-* [Camera triggering](../peripherals/camera.md) via MAVLink out
-* [Camera timing](../peripherals/camera.md#camera_capture) feedback via hotshoe input
+Mapping drones use cameras to capture images at time or distance intervals during surveys.
 
-## Cargo drones and alike: Servos / Outputs
+The following topics show how to connect your camera and set up camera triggering:
+* [Camera triggering](../peripherals/camera.md) from flight controller PWM or GPIO outputs, or via MAVLink.
+* [Camera timing feedback](../peripherals/camera.md#camera_capture) via hotshoe input.
 
-* Servo or GPIO triggering (via RC or via commands)
+The MAVLink commands used in survey missions (and as standalone commands) are:
+* [MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL) - set time interval between captures.
+* [MAV_CMD_DO_SET_CAM_TRIGG_DIST](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_DIST) - set distance between captures
+* [MAV_CMD_DO_TRIGGER_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_TRIGGER_CONTROL) - start/stop capturing (using distance or time, as defined using above messages).
+
+## Cargo Drones ("Actuator" Payloads)
```

Comment:
@bkueng @LorenzMeier  We identify mapping drones (camera for triggering), cargo drones (actuator based control for cargo drop or winches or whatever). 

I was thinking maybe adding Surveillance drones and talk about MAVLink cameras? Anything else?

Reply:
Similar requirements, but search and rescue?

Reply:
Thanks @bkueng 
I added a section for that.